### DISCUSSION TITLE
Custom Attributes Checkbox Field

### DISCUSSION THREAD

**Discussion by bitmakers** [woocommerce/woocommerce#43007] [WooCommerce New Product Editor]
I'm currently using the **"woocommerce_after_product_attribute_settings"** hook to include a custom checkbox field for attributes in the **"Product data > Attributes"** tab:

![attributes-custom-checkbox](https://github.com/woocommerce/woocommerce/assets/4325328/cd96e9ba-e19d-472d-a44a-951cd8dcfac0)

How can I replicate this with the new product editor?

**Comment by mattsherman** [comment]
Hi @bitmakers !

Thanks for asking about this. We do not currently provide a means to accomplish this in the new product editor. I've made a note of this so that we can address it.
